gp-templates/translations.php

```diff
@@ -75,19 +75,34 @@
 	<?php
 	$filter_links = array();
 
+	$filters_and_sort = array_merge( array_filter( $filters ), array_filter( $sort ) );
+
+	$all_filters = array(
+		'status' => 'current_or_waiting_or_fuzzy_or_untranslated',
+	);
+
+	array() === array_diff( $all_filters, $filters_and_sort ) || array() === $filters_and_sort ? $extra_classes = array( 'class' => 'filter-current' ) : $extra_classes = array();
```

Comment:
That's still not readable. Let's put the result of the `array() === array_diff( $all_filters, $filters_and_sort ) || array() === $filters_and_sort` in a `$is_current` variable and pass `$is_current ? array( 'class' => 'filter-current' ) : array();` to `gp_link_get()` instead of `$extra_classes`.